src/js/password-strength.js

```diff
@@ -0,0 +1,30 @@
+import { Component } from 'react';
+
+class PasswordCheck extends Component {

```

Comment:
`PasswordCheck` is a bad name, it doesn't say anything about what component does. also file name doesn't match class name inside. rename class to PasswordStrength and move into `components/utils`